core/src/test/java/org/elasticsearch/transport/netty/NettyTransportIT.java

```diff
@@ -140,6 +142,8 @@ protected String handleRequest(Channel channel, StreamInput buffer, long request
                             if (reg == null) {
                                 throw new ActionNotFoundTransportException(action);
                             }
+                            transport.inFlightRequestsBreaker.addEstimateBytesAndMaybeBreak(messageLengthBytes, "Node in-flight requests");
```

Comment:
This code went slightly out of date anyway over time and I tried to remove as much of it as possible.